Documentation/operations/troubleshooting_clustermesh.rst

```diff
@@ -63,8 +63,8 @@ you may perform the following steps to troubleshoot ClusterMesh issues.
 
  #. Validate that ClusterMesh is healthy running ``cilium status --all-clusters`` inside each Cilium agent::
 
-        ClusterMesh:   1/1 clusters ready, 10 global-services
-           k8s-c2: ready, 3 nodes, 8 identities, 10 services, 0 failures (last: never)
+        ClusterMesh:   1/1 remote clusters ready, 10 global-services
+           k8s-c2: ready, 3 nodes, 25 endpoints, 8 identities, 10 services, 0 reconnections (last: never)
```

Comment:
The `25 endpoints, ` change should be reverted (while preserving the `reconnections` one).